cucumber/authenticators_oidc/features/authn_oidc_v2.feature

```diff
@@ -228,17 +195,6 @@ Feature: OIDC Authenticator V2 - Users can authenticate with OIDC authenticator
     Rack::OAuth2::Client::Error
     """
 
-  @negative @acceptance
```

Comment:
although we are removing `nonce` and `state` as configurable variables with PKCE changes - could you still have a bad state or nonce value and if so how do we report that?

Reply:
Good catch!  I'd strongly prefer we test this at the unit level. I'll add this to the OIDC Client specs as that's where an error would originate from if the nonce was bad. I'll also add a test for a bad code challenge (as that is also validated in this part of the code base).

Reply:
That is correct @andytinkham.